permissions/src/iosMain/kotlin/av/AVPermissionHelper.kt

```diff
@@ -62,18 +64,18 @@ internal class AVPermissionHelper<P : Permission>(private val bundle: NSBundle,
             override val declarationName = NSMicrophoneUsageDescription
         }
     }
-    private val authorizationStatus = suspend {
+    private val authorizationStatus: suspend () -> IOSPermissionsHelper.AuthorizationStatus get() = suspend {
         AVCaptureDevice.authorizationStatusForMediaType(type.avMediaType).toAuthorizationStatus()
     }
-    private val timerHelper = PermissionRefreshScheduler(type.permissionManager, authorizationStatus)
+    private var timerHelper = PermissionRefreshScheduler(type.permissionManager, authorizationStatus)
```

Comment:
Don't see why this is var now?

Reply:
@Daeda88 Could you chime in?

Reply:
Wasnt properly cleaned up. Moved back to val